packages/core/src/sass/account-wizard.scss

```diff
@@ -830,7 +830,7 @@
 
 .personal-details-form {
     .dc-radio-group__input {
-        display: unset;
-        appearance: unset;
+        width: 0;
+        height: 0;
```

Comment:
currently we are displaying a styled `span` element as radio button to the user, and the real `radio input` element is kept hidden from user. due to some reason, iOS safari is displaying a shadow of this input element in dark mode. so i just set its size to 0, so its exists in the DOM, but not visible to user. 👻

Reply:
@suisin-deriv @amina-deriv i tried with `visibility: hidden`, and it's not working. it still takes the space and is getting displayed in iOS.